addons/mail/wizard/mail_followers_edit.py

```diff
@@ -0,0 +1,82 @@
+from odoo import fields, models
+from odoo.exceptions import UserError
+from odoo.addons.mail.tools.parser import parse_res_ids
+
+
+class MailFollowersEdit(models.TransientModel):
+    """Wizard to edit partners (or channels) to add/remove them to/from followers list."""
+
+    _name = 'mail.followers.edit'
+    _description = "Followers edit wizard"
+
+    res_model = fields.Char(
+        "Related Document Model", required=True, help="Model of the followed resource"
+    )
+    res_ids = fields.Char("Related Document IDs", help="Ids of the followed resources")
+    operation = fields.Selection(
+        [
+            ("add", "Add"),
+            ("remove", "Remove"),
+        ],
+        string="Operation",
+        required=True,
+        default="add",
+    )
+    partner_ids = fields.Many2many("res.partner", required=True, string="Followers")
+    message = fields.Html("Message")
+    notify = fields.Boolean("Notify Recipients", default=False)
+
+    def edit_followers(self):
+        for wizard in self:
+            res_ids = parse_res_ids(wizard.res_ids, self.env)
+            documents = self.env[wizard.res_model].browse(res_ids)
+            if wizard.operation == "remove":
+                for document in documents:
+                    document.message_unsubscribe(partner_ids=wizard.partner_ids.ids)
```

Comment:
Isn't message_unsubscribe batch enabled ? As there are no notifications, no need to do a per-document update.